programs/voter-stake-registry/tests/program_test/utils.rs

```diff
@@ -32,3 +34,60 @@ pub fn create_signer_key_and_nonce(program_id: &Pubkey, acc_pk: &Pubkey) -> (Pub
 pub fn clone_keypair(keypair: &Keypair) -> Keypair {
     Keypair::from_base58_string(&keypair.to_base58_string())
 }
+
+#[allow(dead_code)]
+pub async fn get_lockup_data(
```

Comment:
ideally drop this, the struct version is much nicer

Reply:
Refactored test_reset_lockup